for-developers/developer-tasks.R

```diff
@@ -22,18 +17,14 @@ gp <-
 goodpractice::results(gp)
 gp
 
-devtools::run_examples(); #dev.off() #This overwrites the NAMESPACE file too
-# devtools::run_examples(, "redcap_read.Rd")
+devtools::run_examples(); # dev.off() # This overwrites the NAMESPACE file too
 test_results_checked <- devtools::test()
-# test_results_checked <- devtools::test(filter = "read_b.*")
```

Comment:
@kuriwaki, consider keeping this line.  There are a bunch of times where I want to isolate and test a single file.  This line gives provides a headstart.